en/docs/fragments/db-advanced-config.md

```diff
@@ -0,0 +1,29 @@
+Apart from the basic configurations specified above, WSO2 Identity Server supports some advanced database configurations as well.
+
+-	`WSO2_IDENTITY_DB` `deployment.toml` configurations.
```

Comment:
can we present this in a better way. such as: `WSO2_IDENTITY_DB` related configurations that should be added <IS_HOME>/repository/conf/deployment.toml file.

Reply:
check all applicable places